cdap-ui/app/cdap/components/RulesEngineHome/RuleBooksTab/RulesBooksTab.scss

```diff
@@ -26,49 +51,61 @@ $create_rulebook_btn_font_color: #cccccc;
   .rule-books-container {
     display: flex;
     flex-wrap: wrap;
-    padding: 10px;
-    // Stretch the parent by 20px on right and left as children have 10px margin
-    // between them and its always 2 column layout
-    margin-left: -20px;
-    margin-right: -20px;
+    justify-content: space-between;
 
     .rule-book {
-      width: calc(50% - 20px);
+      margin: 15px 0 0;
+      height: 204px;
```

Comment:
The 204px is in the specs. Actually I have removed the `min-height`s altogether, since we're not dynamically changing the height of the rulebooks